### ISSUE TITLE
UglifyCSS transform 100% to 10% for flex:1 100% or flex-basis: 100%

### ISSUE THREAD

**Issue by alexandrursu** [fmarcia/UglifyCSS#60]
UglifyCSS transform 100% to 10% when using `flex:1 100%;,` output becomes` flex:1 10%;`.

It works if I use `flex:1 0 100%;`.

**Comment by fmarcia** [comment]
I can't reproduce this:
```shell
$ ./uglifycss --version
uglifycss 0.0.29
$ echo 'a{flex:1 100%;}' | ./uglifycss
a{flex:1 100%}
```
Are you using the latest version?

**Comment by wilker7ribeiro** [comment]
Same here
+1

**Comment by wilker7ribeiro** [comment]
I'm using gulp-uglifycss, that uses uglifycss@^0.0.25, and when uglifying angular-material lib, all my `flex:1 100%;,` becomes `flex:1 10%;` when using 0.0.28

```
$ ./uglifycss --version
uglifycss 0.0.28
$ echo 'a{flex:1 100%;}' | ./uglifycss
a{flex:1 10%}
```
I will test with 0.0.29

**Comment by wilker7ribeiro** [comment]
Issue was introduced in 0.0.28 and fixed with 0.0.29
